kafka-rest/src/test/java/io/confluent/kafkarest/resources/v3/ProduceActionTest.java

```diff
@@ -649,18 +750,26 @@ private static ProduceAction getProduceAction(
             clock),
         chunkedOutputFactory,
         times,
-        0);
+        0,
+        errorSchemaRegistry);
   }
 
   private static ProduceAction getProduceAction(
       ProduceRateLimiters produceRateLimiters,
       ChunkedOutputFactory chunkedOutputFactory,
       int times,
-      int producerId) {
+      int producerId,
+      boolean errorSchemaRegistry) {
     Provider<SchemaManager> schemaManagerProvider = mock(Provider.class);
+    SubjectNameStrategy subjectNameStrategyMock = mock(SubjectNameStrategy.class);
+    SchemaManager schemaManager =
+        new SchemaManagerImpl(Optional.empty(), subjectNameStrategyMock, false);
```

Comment:
Just out of curiosity, can't this be a mock (which will allow `SchemaManagerImpl` to have default visibility instead of `public`? Not pushing at all for changing it though, no matter if a mock would work or not.